cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -533,13 +534,14 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             };
         }
     });
+    
     GradingEditor = AbstractEditor.extend({
         templateName: 'grading-editor',
         className: 'edit-settings-grading',
 
         afterRender: function () {
             AbstractEditor.prototype.afterRender.call(this);
-            this.setValue(this.model.get('format'));
+            this.setValue(this.model.get('format') || 'notgraded');
```

Comment:
Sorry, I can't confirm. What is the case when this.model.get('format') is undefined?